### DISCUSSION TITLE
SelectInput wierd behavior

### DISCUSSION THREAD

**Discussion by jwhoami** [filamentphp/filament#10167] [Help]
I don´t know how to describe the issue hence the weird title.

So, I have the select column (filament version 3.1.15)

`

      Forms\Components\Select::make('stage')

        ->label(__('models/folio.fields.stage'))

        ->required()

        ->selectablePlaceholder(false)

        ->options(Config::make()->getp('opciones.folios-etapas', [])),
`

If the selectablePlaceholder is true, the first option is automatically selected as expected.
However it is a required field and when I submit the form, a validation error is thrown as seen in the image
even though an option is indeed selected as seen in the image.

![i1](https://github.com/filamentphp/filament/assets/74318608/47a00924-7ba5-4e6d-b121-2d43bae313f1)

If I set the selectablePlaceholder to true or remove it, everything works as expected.

Any clues...

Thanks

**Comment by leandrocfe** [comment]
Could you share the whole code please?
